src/index.js

```diff
@@ -3,11 +3,11 @@ import more from './images/bin.png';
 // import add from './modules/add.js';
 // import removeTask from './modules/remove.js';
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.